docs/unified/novacustom/initial-deployment.md

```diff
@@ -44,29 +44,29 @@ devices.
         === "NV4x"
             ![nv4x chips](../../images/nv4x_board_chips.jpg)
 
-    1. Disconnect the primary battery. (1)
-    1. Disconnect the CMOS battery. (2)
-    1. Attach a WSON-8 probe to the SPI flash chip. (3)
+    2. Disconnect the primary battery. (1)
```

Comment:
@wessel-novacustom There is no need to change that. In rendered form, all numbering lists are automatically converted from `1.` to correct numbers.